config/webpack.dev.js

```diff
@@ -15,5 +15,8 @@ module.exports = merge(common, {
   plugins: [
     new webpack.NamedModulesPlugin(),
     new webpack.HotModuleReplacementPlugin(),
+    new webpack.DefinePlugin({
+      'process.env.PP_DEV_MOCK_VIEWER': true,
```

Comment:
`process.env.FOO` is usually used for Node.js stuff, I think we can go with e.g. `__PP_DEV_MOCK_VIEWER__`